README.md

```diff
@@ -1,7 +1,9 @@
 # Qiskit Runtime
-
 [![License](https://img.shields.io/github/license/Qiskit/qiskit-terra.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)
 
+> **Warning**: This repository was only temporarily used and never published. It is archived and no longer maintained. Please use https://github.com/qiskit/qiskit-ibm-runtime instead.
```

Comment:
It was published here https://qiskit.org/documentation/partners/qiskit_runtime/
```suggestion
> **Archived**: This repository has been merged with [qiskit-ibm-runtime](https://github.com/qiskit/qiskit-ibm-runtime) to keep documentation closer to code. This repository is archived and no longer maintained. Please use [qiskit-ibm-runtime](https://github.com/qiskit/qiskit-ibm-runtime) instead.
```

Reply:
Thanks for the clarification, @rathishcholarajan. Shows that it's definitely a cause for confusion because I did no even remember what `Qiskit-Partners/qiskit-runtime` was used for.